### PR TITLE
feat: add store notification content by env variable

### PR DESCRIPTION
**Pull request by djabarovgeorge**
### What change does this PR introduce? 

add the ability for the user to choose if he wants to store the notification content or not.

### Why was this change needed?

in order to reduce the redundant stored in the persistent stores.

### CODE REVIEW

**Comment by djabarovgeorge** [comment]
Should we add the un-store flag to the createExecutionDetails as well? (we pass the content on DetailEnum.MESSAGE_CREATED event)

File: apps/api/src/.example.env [hunk 1]
```diff
@@ -38,3 +38,5 @@ VERCEL_REDIRECT_URI=http://localhost:4200/auth/login
 VERCEL_BASE_URL=https://api.vercel.com
 
 FF_IS_TOPIC_NOTIFICATION_ENABLED=true
+
+STORE_NOTIFICATION_CONTENT=true
```
Comment (scopsy) [inline apps/api/src/.example.env:42]: Can we make it false by default? Also adding this to env validator?

Comment (djabarovgeorge) [reply apps/api/src/.example.env:42]: Do you mean on the .example.env?

Comment (scopsy) [reply apps/api/src/.example.env:42]: No in this file: apps/api/src/config/env-validator.ts where the schema is defined for env variables and defaults also provided

Comment (djabarovgeorge) [reply apps/api/src/.example.env:42]: Oh got ya, I will update shortly! 👍

**Comment by scopsy** [comment]
@djabarovgeorge let's remove this from execution detail aswell 🙏

**Review by scopsy — APPROVED** [review]

**Pull request merged**